packages/piavpn/piavpn.pacscript

```diff
@@ -0,0 +1,22 @@
+name="piavpn"
+breaks="${name}-git"
+repology=("project: ${name}")
```

Comment:
![](https://0x0.st/oqSX.png)

Won't work as the package in repology uses a different format of the same version, IDK if @Henryws could device a method to automatically convert the repology version format to a usable one for this pacscript.

Reply:
I could, but I don't know how that would work in the context of the repology variable

Reply:
> url="https://installers.privateinternetaccess.com/download/pia-linux-${version}.run"

As you can see the current `version` variable is being directly embedded in the `url` link. So I need you to device a method to automatically convert the version in the repology screenshot to a format that can be used by this link.

I mean just assume the `version` in the pacscript is the repology version, and then write code to make that `version` variable usable in the `url`.